blueprints/public.py

```diff
@@ -1,12 +1,16 @@
-from flask import Blueprint, render_template, request
+from flask import Blueprint, render_template, request, send_file, make_response
+from modules.db import db
+from bson.objectid import ObjectId
+import json
+from icalendar import Calendar, Event
 
 public = Blueprint('public', __name__, template_folder='../templates', static_folder='../static/build/')
 
 @public.route('/')
 def public_index():
     return render_template('home.html')
 
-@public.route('/<page>')
+@public.route('/p/<page>')
```

Comment:
Why did you add `/p/`?

Reply:
oops I forgot to get rid of that. I was just making sure that routing worked the way I thought it did